plugins/trivy/pkg/image/scan.go

```diff
@@ -115,7 +115,7 @@ func ScanImageFile(imagePath string, imageID string, tempDir string) ([]models.V
 		os.Remove(reportFile)
 	}()
 
-	report := []models.VulnerabilityList{}
+	report := models.TrivyResults{}
```

Comment:
mmh why did you need to wrap this to  new struct?

Reply:
Added a Results tag for new trivy report schema version